xc-admin/packages/xc-admin-common/src/multisig.ts

```diff
@@ -0,0 +1,80 @@
+import { PublicKey } from "@solana/web3.js";
+import Squads, {
+  DEFAULT_MULTISIG_PROGRAM_ID,
+  getIxPDA,
+  getTxPDA,
+} from "@sqds/mesh";
+import { InstructionAccount, TransactionAccount } from "@sqds/mesh/lib/types";
+import BN from "bn.js";
+import lodash from "lodash";
+
+export async function getActiveProposals(
+  squad: Squads,
```

Comment:
minor comment: it may simplify things to put these functions on a class that holds common arguments. (e.g., the class can have a Squads member so you don't need to keep passing it in.)

Reply:
(you can also put config parameters in the class like the multisig program id so the code is more flexible)

Reply:
future PR